test/DisplayBoardTests.cpp

```diff
@@ -0,0 +1,88 @@
+//added for testing private methods
+//acccording to google test guide
+#include "DisplayBoard.cpp"
+#include "json.hpp"
+#include <gtest/gtest.h>
+
+
+struct DisplayBoardTests : public ::testing::Test
+{
+};
+
+TEST_F(DisplayBoardTests, checkDrawPaddingFor2Elements)
+{
+    std::string comparePad = "    ";
+    std::string padding = drawPadding(2);
```

Comment:
As I already told you, please don't test private functions. Everything shall be tested via drawBoard which should be public

Reply:
Removed tests for private functions.